spec/spec_helper.rb

```diff
@@ -223,6 +223,13 @@ def request( flickraw, type, method, params )
   end
 end
 
+VCR.configure do |config|
+  config.allow_http_connections_when_no_cassette = true
```

Comment:
this config is by default `false` and was causing build errors b/c there were other specs making actual http calls.